tests/integration/annotation_import/conftest.py

```diff
@@ -223,11 +223,12 @@ def entity_inference(prediction_id_mapping):
 @pytest.fixture
 def segmentation_inference(prediction_id_mapping):
     segmentation = prediction_id_mapping['superpixel'].copy()
-    segmentation.update(
-        {'mask': {
-            'instanceURI': "sampleuri",
+    segmentation.update({
+        'mask': {
+            'instanceURI': "https://picsum.photos/200/300",
```

Comment:
This potentially could result in a failed import. If we run the tests enough picsum starts limiting us. Also if there are no [0,0,0] pixels then the import is unsuccessful.